### PR TITLE
fix: fixing grammar mistakes and misplelling in README files

### PR DESCRIPTION
**Pull request by felipe300**
### Changes

- Change misspelling words and change some grammatical mistakes in README.md files

### Testing
Checked manually in my local environment, with no problems

### Docs
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Para poder tener una buena idea de como usar el lenguaje en la vida real, hemos
 
 ## 💜 ¡Hola, soy Ramón!
 
-Soy ingenerio de software, developer relations e instructor de programación viviendo en Viena, Austria. Llevo más de diez años ayudando a startups y compañias realizar sus apps o mantenerlos. ¡Les he enseñado a programar a más de 10 mil personas por todo el mundo!
+Soy ingenerio de software, developer relations e instructor de programación viviendo en Viena, Austria. Llevo más de diez años ayudando a startups y compañias realizar sus apps o a mantenerlas. ¡Les he enseñado a programar a más de 10 mil personas por todo el mundo!
 
-Mi motivador principal son las comunidades tech. Dedico una gran mayoría de mi tiempo ayudando a gente empezando o avanzada en sus carrera, y organisando conferencias, talleres, meetups y otros eventos con la meta de elevar a otros!
+¡Mi motivador principal son las comunidades tech. Dedico una gran catidad de mi tiempo ayudando a gente empezando o está avanzando en sus carrera; y organisando conferencias, talleres, meetups y otros eventos con la meta de elevar a otros!
```
Comment (hola-soy-milk) [inline README.md]:
```suggestion
¡Mi motivación principal son las comunidades tech. Dedico una gran cantidad de mi tiempo ayudando a gente empezando o está avanzando en sus carrera; y organisando conferencias, talleres, meetups y otros eventos con la meta de elevar a otros!
```

File: leccion-04-funciones/README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Pero ¿qué pasa cuando no definimos un tipo de retorno? En JavaScript, una func
 console.log(typeof publicarAdicion(1, 2))); // => undefined
 ```
 
-Sin embargo, `undefined` es un valor, no un tipo! En este caso, podemos definir el tipo de retorno de una función que no devuelve nada como `void`:
+Sin embargo, `undefined` es un valor, !no un tipo! En este caso, podemos definir el tipo de retorno de una función que no devuelve nada como `void`:
```
Comment (hola-soy-milk) [inline leccion-04-funciones/README.md]:
```suggestion
Sin embargo, `undefined` es un valor, ¡no un tipo! En este caso, podemos definir el tipo de retorno de una función que no devuelve nada como `void`:
```

File: leccion-06-genericos/README.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ En esta lección nos encontramos con el próximo concepto más grande que introd
 
 ## 🐾 Primeros Pasos
 
-### ¿Qué son genéricos?
+### ¿Qué son los genéricos?
 
-Si has leido o visto ejemplos de TypeScript, seguro que has visto el tipo `T`. Este se llama un genérico. Nos permite definir funciones, clases u otras que funcionan con cualquier tipo. Por ejemplo:
+Si has leido o visto ejemplos de TypeScript, seguro que has visto el tipo `T`. a esto se le llama "genérico". Nos permite definir funciones, clases u otros que funcionan con cualquier tipo. Por ejemplo:
```
Comment (hola-soy-milk) [inline leccion-06-genericos/README.md]:
```suggestion
Si has leido o visto ejemplos de TypeScript, seguro que has visto el tipo `T`. A esto se le llama "genérico". Nos permite definir funciones, clases u otros que funcionan con cualquier tipo. Por ejemplo:
```

File: leccion-08-metodo-constructor/README.md [hunk 4]
```diff
@@ -54,19 +54,19 @@ class Perro {
 }
 ```
 
-Cambiémosla para que nombre sea una propiedad pública:
+Cambiémos elcódigo para que `nombre` sea una propiedad pública:
```
Comment (hola-soy-milk) [inline leccion-08-metodo-constructor/README.md]:
```suggestion
Cambiémos el código para que `nombre` sea una propiedad pública:
```

File: leccion-11-ts-estricto-noImplicitAny/README.md [hunk 5]
```diff
@@ -70,7 +70,7 @@ En esta unidad, vamos a activar un par de estas reglas e ir arreglando el códig
 
 ### `noImplicitAny`
 
-Esta regla no permite tener inferencia de tipos. Por ejemplo:
+Esta regla nos permite tener inferencia de tipos. Por ejemplo:
```
Comment (hola-soy-milk) [inline leccion-11-ts-estricto-noImplicitAny/README.md]:
```suggestion
Esta regla bloquea tener inferencia de tipos. Por ejemplo:
```

**Review by hola-soy-milk — COMMENTED** [review]
Thank you so much for this, @felipe300!!

I made some suggestions below. What do you think?

File: README.md (hunk 1 above)
Comment (felipe300) [reply README.md]: I would change:
- Mi motivador principal 
+ Mi moticavión principal

except that all looks good to me

Comment (hola-soy-milk) [reply README.md]: Perfect, thank you!

**Comment by hola-soy-milk** [comment]
With everything resolve, this looks good to merge. Thank you once again, @felipe300! Much appreciated 💜

**Pull request merged**